lib/jwk/keystore.js

```diff
@@ -395,6 +395,11 @@ var JWKStore = function(registry, parent) {
     value: function(props, local) {
       props = props || {};
 
+      // workaround for issues/109
+      if (typeof props.kid === "number") {
```

Comment:
I agree, but non-null non-string targets also undefined, changing it to `"undefined"`, which is definitely not desired. So it has to be non-null, non-undefined and non-string, like so, will that be acceptable?

```js
if (props.kid !== undefined && props.kid !== null && typeof props.kid !== "string") {
  props.kid = String(props.kid);
}
```
and just `props.kid != null` won't cut it since then it won't cover number 0 ;)